### PR TITLE
Docs: set the correct version for project info

### PR DESCRIPTION
**Pull request by ennru**
## Purpose

Make sure the project info gets hold of the correct version number.

## References

Fixes https://github.com/lightbend/sbt-paradox-project-info/issues/15

### CODE REVIEW

**Comment by ennru** [comment]
I find it important to let the users discover the versioned API docs here. That is part of my idea behind the project info section.
If we don't want to show patch version API docs to anybody, we should always just keep the latest version per minor instead.

File: project/Common.scala [hunk 1]
```diff
@@ -6,10 +6,12 @@ package akka
 
 import sbt._
 import Keys._
+import com.lightbend.paradox.projectinfo.ParadoxProjectInfoPluginKeys._
 
 object Common extends AutoPlugin {
   override def trigger: PluginTrigger = allRequirements
   override lazy val projectSettings = Seq(
+  projectInfoVersion := (if (isSnapshot.value) "snapshot" else version.value),
```
Comment (jrudolph) [inline project/Common.scala]: Indentation:

```suggestion
    projectInfoVersion := (if (isSnapshot.value) "snapshot" else version.value),
```

**Comment by ennru** [comment]
This value is used to create the API links in the Project Info sections of the docs (it reverts #2857).
I know @raboof and I might have different opinions on what to link to, without this change Akka HTTP 10.1.x documentation will link to 10.2.x API docs once they are considered "current", which I find wrong. With this change they will point to the exact same version of API docs as the docs they belong to.

**Comment by jrudolph** [comment]
> This value is used to create the API links in the Project Info sections of the docs (it reverts #2857).
> I know @raboof and I might have different opinions on what to link to, without this change Akka HTTP 10.1.x documentation will link to 10.2.x API docs once they are considered "current", which I find wrong. With this change they will point to the exact same version of API docs as the docs they belong to.

Hmm, for me it only shows the project info link for `current` right now, so it probably wouldn't help.

That said, for API docs, I agree, that putting the right version there makes sense (but I'm more concerned with usability of the site vs. SEO).

We also have https://doc.akka.io/docs/akka-http/10.1.11/reference.html which also links to the right version of the API docs (and is more likely being used than the collapsed project info box).

**Comment by jrudolph** [comment]
> Hmm, for me it only shows the project info link for `current` right now, so it probably wouldn't help.

Do we have an explanation for that?

**Comment by ennru** [comment]
With these changes, you should not get `current`, either `snapshot` or the fixed version during releasing.

**Comment by jrudolph** [comment]
Hmm, locally built the links now go to `https://doc.akka.io/api/akka-http/0.1.0-SNAPSHOT/akka/http/scaladsl/index.html` which doesn't seem right.

**Comment by ennru** [comment]
Are you sure you reloaded sbt?

**Comment by jrudolph** [comment]
> Are you sure you reloaded sbt?

Hmm, seems to work for me today :)

**Pull request merged**